app/drivers/zephyr/kscan_gpio_matrix.c

```diff
@@ -51,6 +51,7 @@ static int kscan_gpio_config_interrupts(struct device **devices,
 }
 #endif
 
+
```

Comment:
Revert?

Reply:
I'm not sure I follow the question. Can you elaborate?